pkg/v1/sdk/capabilities/controllers/capability_controller.go

```diff
@@ -30,15 +30,15 @@ type CapabilityReconciler struct {
 //+kubebuilder:rbac:groups=run.tanzu.vmware.com,resources=capabilities/status,verbs=get;update;patch
 
 // Reconcile reconciles a Capability spec by executing specified queries.
-func (r *CapabilityReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
-	ctx, cancel := context.WithTimeout(context.Background(), contextTimeout)
+func (r *CapabilityReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	ctxCancel, cancel := context.WithTimeout(context.Background(), contextTimeout)
```

Comment:
Same as feature gates reconciler. Use the passed in `ctx` instead of `context.Background()`.